connectivity/mbedtls/platform/inc/timing_alt.h

```diff
@@ -24,11 +24,9 @@
 #include "mbedtls/timing.h"
 #if defined(MBEDTLS_TIMING_ALT)
 
-#include <time.h>
-
 struct mbedtls_timing_hr_time
 {
-    struct timeval start;
+    unsigned long start;
```

Comment:
Why not make `gettimeofday()` more precise (to ms)? Wouldn't that benefit more use cases?

Reply:
OK, good to follow up on later. If we add `alarm()` and `signal()` to Mbed OS, we may be able to use the default timing implementation from Mbed TLS and not have to implement our own, given we have a `gettimeofday()` precise within 100ms as the timing self-test requires.